app/helpers.rb

```diff
@@ -0,0 +1,23 @@
+module Helpers
+
+  def current_user
+    @current_user ||= User.get(session[:user_id])
+  end
+
+  def search_available_spaces(date_from)
```

Comment:
future implementation to refactor out into private methods and include ending date